requests_kerberos/kerberos_.py

```diff
@@ -86,70 +84,75 @@ class HTTPKerberosAuth(AuthBase):
     object."""
     def __init__(
             self, mutual_authentication=REQUIRED,
-            service="HTTP", delegate=False):
+            service="HTTP", delegate=False, force_preemptive=False):
```

Comment:
This new public setting is worth explaining in the readme file. Setting this to true is what triggers the desired always-reauthenticate-per-request behaviour, correct? Is it sensible to disable the new behaviour by default?